### ISSUE TITLE
Firefox support

### ISSUE THREAD

**Issue by rstacruz** [lra/mackup#633]
Is there any reason why firefox support isn't available?

**Comment by lra** [comment]
Syncing is builtin no?

**Comment by rstacruz** [comment]
Fair enough! I guess things like accounting for temp files and caches can be hairy

**Comment by rafaelliu** [comment]
I'd like to propose this to be re-opened. VS Code also has sync, and there's still a mackup plugin.

Firefox does support sync but some of us use a combination of mackup, dot files and git. It would be nice to have it all centralized, versioned and automated.

Other reasons Firefox Sync may not be an option: self-host philosophy, corporate managed laptops, other firefox-based distros, etc

**Comment by lra** [comment]
That's understandable, but in order to do so, I'd have 1st to officially support per-app sync, to not enable sync by default on everyone.

We can reopen this once this is done.

Thanks.
